submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/model/SubmissionRequestModel.kt

```diff
@@ -19,7 +16,18 @@ data class DocSubmissionRequest(
     val draftKey: String?,
     val status: RequestStatus,
     val submission: DBObject,
-    val totalFiles: Int = 0,
-    val currentIndex: Int = 0,
-    val modificationTime: Instant = Instant.now(),
+    val totalFiles: Int,
+    val currentIndex: Int,
+    val modificationTime: Instant,
+)
+
+@Document(collection = "submission_request_files")
+data class DocSubmissionRequestFile(
+    @Id
+    val id: ObjectId,
+    val index: Int,
+    val accNo: String,
+    val version: Int,
+    val path: String,
+    val file: DBObject,
```

Comment:
I think we should use here DocFile rather that DbObject in that way representation has garantee to have same structure that a submission file (also you can re use its mapping)

Reply:
Please check my previous comment